broker_app.py

```diff
@@ -118,17 +118,23 @@ def get_spreadsheet(job_id: str):
 
 def create_app():
     app = Flask(__name__, static_folder='static')
-    app.SPREADSHEET_STORAGE_DIR = os.environ.get('SPREADSHEET_STORAGE_DIR')
-    app.SPREADSHEET_UPLOAD_MESSAGE = "We’ve got your spreadsheet, and we’re currently importing and validating the data. \
-Nothing else for you to do - check back later."
+    app.SPREADSHEET_STORAGE_DIR = os.getenv('SPREADSHEET_STORAGE_DIR')
+    app.SPREADSHEET_UPLOAD_MESSAGE = "We’ve got your spreadsheet, and we’re currently " \
```

Comment:
Also this.

Reply:
you can tag multiple lines in GitHub comments, i should show you at some point